### PR TITLE
CHORE: update web API project template .NET Core 2.2 > 3.1

### LINKED ISSUE #87: Switch Web API template to .NET Core 3.1
Switch Web API template to .NET Core 3.1

### PR DESCRIPTION
**Pull request by stijnmoreels**
Update the web API project template with .NET Core 3.1 SDK.

Closes #87

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Can you also update to Swashbuckle 5+ please?

**Comment by tomkerkhove** [comment]
Let's change the following in `Configure` as well:
- `SwaggerEndpoint`  to `/api/v1/docs.json`
- `RoutePrefix` to `api/docs`

Swagger is now called OpenAPI as of 3.x so better get rid of Swagger

**Comment by tomkerkhove** [comment]
Too fast @stijnmoreels 😁

Here is a nice overview of what has changed: https://github.com/domaindrivendev/Swashbuckle.AspNetCore/releases/tag/v5.0.0

File: src/Arcus.Templates.Tests.Integration/WebApi/Health/v1/TimeSpanConverter.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Arcus.Templates.Tests.Integration.WebApi.Health.v1 
+{
+    /// <summary>
+    /// JSON converter to parse <see cref="TimeSpan"/> values.
+    /// Fixing issue: https://github.com/JamesNK/Newtonsoft.Json/issues/2189.
+    /// </summary>
+    public class TimeSpanConverter : JsonConverter<TimeSpan>
```
Comment (tomkerkhove) [inline src/Arcus.Templates.Tests.Integration/WebApi/Health/v1/TimeSpanConverter.cs]: Is there no built-in way? That would be strange.

Comment (stijnmoreels) [reply src/Arcus.Templates.Tests.Integration/WebApi/Health/v1/TimeSpanConverter.cs]: See issue. Apperently a problem since .NET Core 3.0.

Comment (tomkerkhove) [reply src/Arcus.Templates.Tests.Integration/WebApi/Health/v1/TimeSpanConverter.cs]: Looked over that one, sorry.

**Comment by tomkerkhove** [comment]
Can we also rename Swagger in our docs to OpenAPI please?
https://templates.arcus-azure.net/features/web-api-template#features

**Comment by tomkerkhove** [comment]
Did you manually run it to see if everything is fine as well?

**Comment by stijnmoreels** [comment]
You manually run the swagger and health? Yes.
The tests are more thorough about all the other stuff.

**Comment by tomkerkhove** [comment]
> You manually run the swagger and health? Yes.
> The tests are more thorough about all the other stuff.

Tests don't always cover everything ;)

**Pull request merged**